server/src/main/java/io/unitycatalog/server/service/AuthService.java

```diff
@@ -185,9 +185,12 @@ public HttpResponse logout(HttpRequest request) {
             authorizationCookie -> {
               Cookie expiredCookie = createCookie(AuthDecorator.UC_TOKEN_KEY, "", "/", "PT0S");
               ResponseHeaders headers =
-                  ResponseHeaders.of(
-                      HttpStatus.OK, HttpHeaderNames.SET_COOKIE, expiredCookie.toSetCookieHeader());
-              return HttpResponse.of(headers);
+                  ResponseHeaders.builder()
+                      .status(HttpStatus.OK)
+                      .add(HttpHeaderNames.SET_COOKIE, expiredCookie.toSetCookieHeader())
+                      .contentType(MediaType.JSON)
+                      .build();
+              return HttpResponse.of(headers, HttpData.ofUtf8("{}"));
             })
         .orElse(HttpResponse.of(HttpStatus.OK));
```

Comment:
Hmm.. maybe we should send an empty JSON here too, just to be consistent.

Reply:
make sense have  made the changes